_dev/coding-style.md

```diff
@@ -6,8 +6,65 @@ description: The preferred style for code committed to NeoMutt
 
 # Coding Style
 
-Every project has its own coding style, this one is no exception.
-However, I haven't written down the rules.  For now, ...
+## at the moment (pre-fork)
 
-- Trailing whitespace is **evil**
+Generally speaking, we don't want to diverge much from upstream mutt. We're
+still merging patches in - and if we didn't follow upstream's coding style,
+merging would be much harder.
 
+That means:
+
+* put your the braces of functions in new lines, like so:
+
+```c
+struct q_class_t
+{
+  int length;
+  int index;
+  int color;
+  char *prefix;
+  struct q_class_t *next, *prev;
+  struct q_class_t *down, *up;
+};
+```
```

Comment:
if you say *put your the braces of functions in new lines, like so* it'd be better to show a function thereafter, not a `struct`. It'be actually be more exact to state: 

* all blocks are surrounded by braces on a single line, *e.g.*:

``` c
struct example_t
{
  // …
}

void example (void)
{
  if (1)
  {
    // …
  }
  // …
}
```